using System;
namespace Day_5
{
					
public class Program
{
	public static void Main()
	{
		int a=5;
		//int c= a/0;
		int[] arr=new int[2];
		try
		{
		Console.WriteLine(arr[2]);
		}
		catch(Exception e)
		{
		  Console.WriteLine("Some exception"+e.Message);
		}
		Console.ReadLine();
	  }
	}
}